build.gradle

```diff
@@ -320,6 +320,10 @@ project(":samza-yarn_$scalaVersion") {
     compile "org.scala-lang:scala-compiler:$scalaLibVersion"
     compile "org.codehaus.jackson:jackson-mapper-asl:$jacksonVersion"
     compile "commons-httpclient:commons-httpclient:$commonsHttpClientVersion"
+    compile "org.apache.httpcomponents:httpcore:$httpClientVersion"
+    compile "org.apache.httpcomponents:httpclient:$httpClientVersion"
+    compile "org.bouncycastle:bcpkix-jdk15on:$bouncyCastleVersion"
```

Comment:
this package is an open source one for cryptography, we use the functions from it to generate the x500Name, to sign the x500Name, and to generate CSR.